docs/contributing/how-to-contribute.md

```diff
@@ -6,7 +6,7 @@ We want contributing to Gatsby to be fun, enjoyable, and educational for anyone
 
 ## Not sure how to start contributing?
 
-If you are worried or don't know where to start, check out our [where to participate](/contributing/where-to-participate/) page. You can reach out with questions to [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) or [@GatsbyJS](https://twitter.com/gatsbyjs) on Twitter, and anyone from the Gatsby team on [Discord](https://gatsby.dev/discord). You can also [submit an issue](/contributing/how-to-file-an-issue/) and a maintainer can give you guidance!
+If you are worried or don't know where to start, check out our [where to participate](/contributing/where-to-participate/) page. You can reach out with questions to [@GatsbyJS](https://twitter.com/gatsbyjs) on Twitter, and anyone from the Gatsby team on [Discord](https://gatsby.dev/discord). You can also [submit an issue](/contributing/how-to-file-an-issue/) and a maintainer can give you guidance!
```

Comment:
Should we point it to @AskGatsbyJS instead, since that's more of a support channel and gets less volume?

Reply:
Yep, that's a good point. I'll update that - thanks!